src/main/scala/io/iohk/ethereum/db/storage/pruning/RangePrune.scala

```diff
@@ -0,0 +1,15 @@
+package io.iohk.ethereum.db.storage.pruning
+
+import io.iohk.ethereum.utils.Logger
+
+trait RangePrune extends Logger {
+  def pruneBetween(start: BigInt, end: BigInt, pruneFn: BigInt => Int): PruneResult = {
```

Comment:
- `pruneFn` is a function that given a block tag, prunes and returns how many nodes has deleted
- `io.iohk.ethereum.db.storage.pruning.PruningMode#PruneFn` is a function that based on the last pruned block number tag and the latest block number, prunes the data and returns new last pruned block and the amount of nodes removed.

Maybe, after the refactor suggested [here](https://github.com/input-output-hk/etc-client/pull/253/files#r128592298) I can improve that.